### DISCUSSION TITLE
Add roadmap

### DISCUSSION THREAD

**Discussion by omridevk** [chakra-ui/panda#1312] [General]
### Subject

I would like to see a roadmap documenation page reflecting the future of the project and maybe even a milestone for v1 release.

### Description

Really really awesome library, was wondering if there's a clear roadmap on features you wish to add in the future?

**Comment by itsjavi** [comment]
I was going to open a similar discussion, because I think it's important for the community. 
@segunadebayo

I've been using Panda since v0.1x.x and at some point I stopped using it because I couldn't cope with the amount of changes that were introduced, breaking my code many times.

I would like to know what are the plans for a stable release?

I love the DX of Panda and it has no comparison (not even StyleX can compete IMHO), but I think developers need to know what are the plans on a more stable release (included a feature-freeze).

Thank you for this amazing project.

**Reply by cgatian** [reply]
Exactly. I would love to know more about the roadmap and when a 1.0 can be expected.

**Comment by omridevk** [comment]
By the way, found this:
https://www.adebayosegun.com/blog/panda-v1-roadmap

**Reply by omridevk** [reply]
Thanks @segunadebayo , you're awesome!

**Comment by marviobezerra** [comment]
No mention to react native!? I know that it’s not that simple, but for sure it would be a huge improvement.
